pkg/prompts/prompts.go

```diff
@@ -32,8 +35,18 @@ func RunPromptsFromConfigWithSkipsIO(config *config.DraftConfig, varsToSkip []st
 			log.Debugf("Skipping prompt for %s", promptVariableName)
 			continue
 		}
+		if customPrompt.NoPrompt {
+			log.Debugf("Skipping prompt for %s as it has NoPrompt=true", promptVariableName)
```

Comment:
💡 most of these loggers, can we make a `utility` --> `logger funct` and use it as one standard way, i.e. line-35, line-49 and line-39 both tries to capture **same info** just initial string is different.

Reply:
Not sure I follow what you're trying to accomplish with a utility func surrounding the the logger? Can you elaborate more @Tatsinnit? It sounds like an enhancement outside of this change if we do want to do it though.

Reply:
i don't understand what the gains of abstracting the `log.Debugf` function would yield, since it is already a short and concise name that is set through the logrus library. If we added a `utility.Debugf` it would end up being a function that just calls `log.Debugf` transparently right?

I do see that we have redundant text for "Skipping prompt for %s", so I could refactor that out into a constant, but each call to the debug log has a different ending so i would still have to add a custom string in each one

Reply:
Cool, yeah whatever make sense long term, I noticed hence mentioned that we could have one helper funct which can do same thing but we can log function which takes what we are logging and and logging message as standard:

Observation so with in one funct we have these logging - are these for engineering purposes aka us? (or might be as simple as keeping a quick readme as to what we are logging as fyi)

Not in the scope of this PR - but mentioned because we added another logger for same variable and hence worth to fyi as point of discussion. 

thanks heaps.

``` 

log.Debugf("Skipping prompt for %s", promptVariableName)

log.Debugf("Skipping prompt for %s as it has NoPrompt=true", promptVariableName)

log.Debugf("constructing prompt for: %s", promptVariableName)

```

Reply:
that sounds good we can refactor for clarity in another PR